src/site/content/en/progressive-web-apps/app-like-pwas/index.md

```diff
@@ -0,0 +1,438 @@
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  The <a href="https://web.dev/content-indexing-api/">Content Index API</a> allows your application
+  to tell the browser which content of the PWA is available offline.
+  This allows the browser to surface this content outside of the main app.
+  By marking up interesting content in your app as suitable for <a href="https://developers.google.com/search/docs/data-types/speakable">speakable</a>
+  audio playback and by using <a href="https://developers.google.com/search/docs/guides/search-gallery">structured markup</a> in general,
+  you can help search engines and virtual assistants like the Google Assistant present your offerings in an ideal light.
+{% endDetails %}
+
+## Lock screen media control widget
+
+When a podcast episode is playing, the Podcasts app shows a beautiful control widget on the lock screen that features metadata like the episode artwork, the episode title, and the podcast name.
+
+<figure class="w-figure">
+  <img src="./image17.png" alt="iOS media playback widget on the lock screen showing a podcast episode with rich metadata." width="300">
+  <figcaption class="w-figcaption">Media playing in the app can be controlled from the lock screen.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  The <a href="https://web.dev/media-session/">Media Session API</a> lets you specify metadata like artwork, track titles, etc.
+  that then gets displayed on the lock screen, smartwatches, or other media widgets in the browser.
+{% endDetails %}
+
+## Push notifications
+
+Push notifications have become a bit of an annoyance on the web
+(albeit [notification prompts are a lot quieter](https://blog.chromium.org/2020/01/introducing-quieter-permission-ui-for.html) now),
+but if used properly, they can add a lot of value.
+For example, the iOS Podcasts app can optionally notify me of new episodes of podcasts I am subscribed to or recommend me new ones, as well as alert me of new app features.
+
+<figure class="w-figure">
+  <img src="./image12.png" alt="iOS Podcasts app in the 'Notifications' settings screen showing the 'New Episodes' notifications toggle activated." width="300">
+  <figcaption class="w-figcaption">Apps can send push notifications to inform the user about new content.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  The <a href="https://developers.google.com/web/fundamentals/push-notifications">Push API</a>
+  allows your app to receive push notifications so you can notify your users about noteworthy events around your PWA.
+  For notifications that should fire at a known time in the future and that do not require a network connection,
+  you can use the <a href="https://web.dev/notification-triggers/">Notification Triggers API</a>.
+{% endDetails %}
```

Comment:
This section also implies to me that this feature is available on iOS

Reply:
Same as before.